ferveo-python/src/lib.rs

```diff
@@ -66,9 +66,13 @@ pub fn encrypt(
     dkg_public_key: &DkgPublicKey,
 ) -> PyResult<Ciphertext> {
     let rng = &mut thread_rng();
-    let ciphertext =
-        ferveo::api::encrypt(message, aad, &dkg_public_key.0 .0, rng)
-            .map_err(map_py_err)?;
+    let ciphertext = ferveo::api::encrypt(
+        ferveo::api::SecretBox::new(message.to_vec()),
```

Comment:
Just a note, you're having a double allocation here, since `SecretBox` allocates for its contents. But that's something for the future.